### PR TITLE
Update Links

### PR DESCRIPTION
**Pull request by avsavsavs**
This PR updates the view analytics link, and the Coincidence of Wants link

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ export default function Home({ metricsData, siteConfigData }) {
         <div>
           <SectionImage margin={'0 auto -18rem'} height={'68rem'}><img loading="lazy" src="/images/cowBelt.jpg" alt="A fast growing protocol" /></SectionImage>
           <h2>A fast-growing trading protocol</h2>
-          <SubTitle align="center">Trade on CoW Protocol for <br /> better prices, gas cost savings and extra secure MEV protection <ExternalLink href="#">View analytics</ExternalLink></SubTitle>
+          <SubTitle align="center">Trade on CoW Protocol for <br /> better prices, gas cost savings and extra secure MEV protection <ExternalLink href="https://dune.xyz/gnosis.protocol/Gnosis-Protocol-V2">View analytics</ExternalLink></SubTitle>
```
Comment (harisang) [inline pages/index.tsx]: Would also add a dot after the word "protection", so that both issues #31 and #32 are addressed

**Comment by fairlighteth** [comment]
@harisang @avsavsavs Is this still relevant?

**Comment by harisang** [comment]
> 

I would say yes, since the first link that this PR fixes is still broken.

**Comment by fairlighteth** [comment]
@harisang Thank you. Now added the dot -> https://cow-web-git-avsavsavs-patch-2-cowswap.vercel.app/

Closes #31 and #32

**Review by harisang — APPROVED** [review]

**Pull request merged**